examples/temperature-measurement-app/esp32/main/ResponseServer.cpp

```diff
@@ -54,6 +54,13 @@ using namespace ::chip::Inet;
 using namespace ::chip::Transport;
 
 extern const NodeId kLocalNodeId = 12344321;
+namespace chip {
+namespace DeviceLayer {
+namespace Internal {
+extern const uint64_t TestDeviceId = kLocalNodeId; // For chip::DeviceLayer::GetDeviceId
+}
```

Comment:
// namespace Internal

is 'extern' supposed to be here? I think external linkage is the default.